ksqldb-engine/src/test/java/io/confluent/ksql/execution/ExpressionEvaluatorParityTest.java

```diff
@@ -424,7 +424,8 @@ private void runEvaluator(
       RecordProcessingError processingError
           = ((RecordProcessingError) errorMessageCaptor.getValue());
       if (error.get().getMessage() == null) {
-        assertThat(processingError.getException().get().getMessage(), nullValue());
+        assertThat(processingError.getException().get().getMessage(),
```

Comment:
`NullPointerException`s in JDK14+ now have better error messages indicating what caused the NPE. While that's _fantastic_, it causes a few of the tests to fail that compare the expected error message